covalent_ssh_plugin/ssh.py

```diff
@@ -79,6 +84,8 @@ def __init__(
         conda_env: str = None,
         remote_cache: str = "",
         run_local_on_ssh_fail: bool = False,
+        remote_workdir: str = "",
+        create_unique_workdir: Optional[bool] = None,
```

Comment:
If it's bool then let's make it's default to be `True` or `False`.

Reply:
The idea is to get it from the config i.e. `get_config("executors.ssh.create_unique_workdir")` if it's `None` i.e. it is not supplied. When the default is `False`, we don't know if it's `False` intentionally OR perhaps the default is `False` and it's an indication to get from the config...

`None` helps us in this case...

PS -- This is the same for `Local` and `Dask` executors as well...

Reply:
You're right, I was going to ask whether there's a specific reason. This seems reasonable 👍 .